railties/lib/minitest/rails_plugin.rb

```diff
@@ -31,6 +31,10 @@ def self.plugin_rails_options(opts, options)
       options[:color] = value
     end
 
+    opts.on("-s", "--skip-rails-reporter", "Skip Rails' custom Minitest reporter") do |value|
```

Comment:
Since `-s` option conflicts with minitest's option(SEED option), it should avoid.

Reply:
Good call. I'll keep just the full switch (`--skip-rails-reporter`).